Makefile

```diff
@@ -96,6 +96,14 @@ endif
 # Optional file including user-specific settings (HUB, TAG, etc)
 -include .istiorc.mk
 
+# Base directory for installing artifacts
+INSTALL_PREFIX ?= /usr/local
+export INSTALL_PREFIX
+
+define replace_install_prefix
+  find $(1) -type f -exec sed -i 's|{INSTALL_PREFIX}|$(INSTALL_PREFIX)|' {} \;
```

Comment:
ideally those would happen outside of the source tree ?

also, for dockerfile, I think we should leave those unchanged/untouched
(it adds more problems than it would solve, it's good to have dockerfile buildable right away)

Reply:
+1 on having this in a tool or outside of source tree.

Reply:
ack